.github/workflows/build.yml

```diff
@@ -86,6 +86,7 @@ jobs:
           python "$(which firedrake-clean)"
           python -m pip install \
             pytest-cov pytest-timeout pytest-xdist pytest-timeout
+          python -m pip uninstall -y matplotlib
```

Comment:
Why is this here?

Reply:
This is temporary to check tests pass without `matplotlib`. By default `matplotlib` is installed as a dependency of FEMlium I think

Reply:
This is now gone